src/components/Modal/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Button from 'src/components/inputs/Button';
+import { ConfirmationButtons, Message } from './styled';
+
+type ConfirmationModalProps = {
+  confirmText: string;
+  cancelText: string;
+  onConfirm: () => void;
+  onCancel: () => void;
+  message: string;
+};
+
+function ConfirmationModal({
+  confirmText,
+  cancelText,
+  onConfirm,
+  onCancel,
+  message,
+}: ConfirmationModalProps) {
+  return (
+    <>
+      <Message>{message}</Message>
+      <ConfirmationButtons>
```

Comment:
I like to call this type of wrapper - that only contains buttons - for `Controls`, `ControlWrapper` or `ControlContainer` 😄. This is just my opinion, but I think that it better informs the reader about the purpose of the component